src/databricks/labs/ucx/queries/migration/main/02_1_code_compatibility_problems.sql

```diff
@@ -1,5 +1,17 @@
--- viz type=table, name=Workflow migration problems, search_by=path,code,job_name, columns=path,code,message,workflow_id,workflow_name,task_key,start_line,start_col,end_line,end_col
--- widget title=Workflow migration problems, row=2, col=2, size_x=4, size_y=8
+/*
+--title 'Workflow migration problems'
+--overrides '{"spec": {"encodings": {"columns": [
```

Comment:
Added the clickable links by overwriting the `columns` in encodings:


https://github.com/databrickslabs/ucx/assets/5946784/e1ae0954-83f3-4165-88f6-6353935e3aef

Note: it requires to overwrite all columns as the whole list needs to be provided